sky/backends/backend_utils.py

```diff
@@ -380,6 +384,7 @@ def _get_generated_config(cls, autogen_comment: str, host_name: str,
               IdentitiesOnly yes
               ForwardAgent yes
               StrictHostKeyChecking no
+              UserKnownHostsFile=/dev/null
```

Comment:
Curious - do we also need to set `GlobalKnownHostsFile=/dev/null`? [[1]](https://man.openbsd.org/ssh_config#GlobalKnownHostsFile) [[2]](https://www.baeldung.com/linux/ssh-known_hosts-ignore-emporarily#3-permanently-ignore)

Reply:
Ahh, good catch! It seems that `GlobalKnownHostsFile=/dev/null` should be included as well. Thanks!